homeassistant/components/tradfri/sensor.py

```diff
@@ -127,6 +149,12 @@ async def async_setup_entry(
             continue
 
         for description in descriptions:
+            _migrate_old_unique_ids(
```

Comment:
Add a code comment saying when we added this so it's easy to decide when it's old enough so we can remove it.